juju/juju
# LXD Fix for Unknown Request Type When Adding Juju Certificate

## Description of change

- Renames "Client" to "Server" for the new functionality under _container/lxd_ (this is in the first commit, so the second can be read for less noise).
- Relocates certificate concerns from the old LXD client to the new LXD server implementation.
- Removes unnecessary wrapper code.
- Ensures certificates are created with type=client.

## QA steps

Modified and additional unit tests.
System test:
- Remove and reinstall LXD (lazily ensuring no certificates).
- Reinstall and init LXD.
- Ensure bootstrap succeeds.

## Documentation changes

None.

## Bug reference

https://bugs.launchpad.net/juju/+bug/1770051
